tests/unit/admin-store-test.js

```diff
@@ -1,30 +1,23 @@
 import Ember from 'ember';
 import { module, test } from 'qunit';
 import DS from 'ember-data';
-import AdminStore from 'ember-admin/stores/admin';
-import AdminService from 'dummy/services/admin';
+import startApp from '../helpers/start-app';
 
 var oldNamespace, adminStore, adminService;
 var set = Ember.set;
 
-module('Admin Store', {
```

Comment:
It might be an issue for a future PR but the cleanest way to rewrite this test now is to use `ember-qunit`'s `moduleFor`:

```js
import { moduleFor, test } from 'ember-qunit';
...

moduleFor('store:admin', 'Unit | Store | admin', {
  needs: ['service:admin'],
  setup: function() {
    App = startApp();
    adminService = this.container.lookup('service:admin');
    adminStore = this.subject();
  },
  teardown: function() {
    App.registry = App.buildRegistry();
    App.reset();
  }
});
```

Something like this